kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/storage/KieServerStateRepository.java

```diff
@@ -15,9 +15,30 @@
 
 package org.kie.server.services.impl.storage;
 
+import java.util.Properties;
+
+import org.kie.server.api.model.KieServerConfig;
+import org.kie.server.api.model.KieServerConfigItem;
+
 public interface KieServerStateRepository {
 
     void store(String serverId, KieServerState kieServerState);
 
     KieServerState load(String serverId);
+
+    default void populateWithSystemProperties(KieServerConfig config) {
```

Comment:
These methods seem to be just utility methods.
Can be moved to utility class, not cluttering this interface. It doesn't have sense to provide them as functionality of this interface, they don't have a relation to it.

Reply:
Moved those two methods into a new utility class. 

https://github.com/rhtevan/droolsjbpm-integration/blob/041052b96b020cc64bd7bd84a9f09d56ec2defbc/kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/storage/KieServerStateRepositoryUtils.java#L23